langwatch/src/pages/[project]/annotations/[slug].tsx

```
@@ -36,15 +37,22 @@ export default function Annotations() {
         <Heading size="lg">{queue.data?.name}</Heading>
-        <HStack>
+        <HStack position="relative" gap={2}>
           <Text fontSize="sm">Members: </Text>
           {queueMembers?.map((member) => {
             return (
-              <Tooltip key={member.id} content={member.name}>
-                <Avatar.Root size="xs">
-                  <Avatar.Fallback name={member.name ?? ""} />
-                </Avatar.Root>
-              </Tooltip>
+              <Box>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)